pkg/handler/commit.go

```diff
@@ -101,3 +102,87 @@ func (h *commitHandler) GetCommitByMessage(w http.ResponseWriter, r *http.Reques
 
 	_ = json.NewEncoder(w).Encode(resp)
 }
+
+func (h *commitHandler) CommitReleased(w http.ResponseWriter, r *http.Request) {
```

Comment:
nitpick: maybe we can split this method into distinct functions for better readability (if possible)

LGTM otherwise

Reply:
Right right. For loop part was the major logic chunk which can be reused in other functions. I have moved it to a separate function. Thanks!